utils/src/main/kotlin/OkHttpClientHelper.kt

```diff
@@ -105,6 +105,6 @@ object OkHttpClientHelper {
                 .build()
         }
 
-        return client.newCall(request).execute()
```

Comment:
I would prefer if you kept this in another `execute` helper function. Than all but one of the calls could stay unchanged and use the simpler API.

Reply:
I was actually considering that, but turned the idea down because that new `execute` helper function would duplicate most `buildClient` parameters just to pass them through, which I don't find so nice.

Reply:
But ok, I'll do it.

Reply:
Done.

Reply:
Thanks.